### PR TITLE
Fix/range output value

### PR DESCRIPTION
**Pull request by germanbisurgi**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Is backward-compatible?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | #1097

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -299,7 +299,10 @@ export class AbstractTheme {
 
   getRangeOutput (input, startvalue) {
     const output = document.createElement('output')
-    output.value = startvalue || 0
+
+    setTimeout(() => {
+      output.value = startvalue || 0
+    }, 0)
```
Comment (schmunk42) [inline src/theme.js:305]: @germanbisurgi 
Hm, is there no other way than a setTimeout with delay 0? Can't this be bound to an event?

File: src/editors/string.js [hunk 2]
```diff
@@ -213,6 +213,10 @@ export class StringEditor extends AbstractEditor {
       /* it will generate an error trying to append it to the missing parentNode */
       if (this.input.parentNode) this.afterInputReady()
       if (this.adjust_height) this.adjust_height(this.input)
+      const output = this.control.querySelector('output')
+      if (output) {
```
Comment (schmunk42) [inline src/editors/string.js:217]: Are there cases when `output` is empty?
Would
```
this.control.querySelector('output').value = this.input.value
```
be sufficient?

Comment (germanbisurgi) [reply src/editors/string.js:217]: When string has format 'range' there is always an output. But a string doe not have to have always format 'range'

Comment (schmunk42) [reply src/editors/string.js:217]: Question: Wouldn't it be better to check for the `format` rather than an element `output`?

Comment (germanbisurgi) [reply src/editors/string.js:217]: I think checking both is better. Should i update?

**Review by schmunk42 — APPROVED** [review]

**Pull request merged**